code/modules/mob/living/simple_animal/bot/floorbot.dm

```diff
@@ -20,23 +20,29 @@
 
 	var/process_type //Determines what to do when process_scan() recieves a target. See process_scan() for details.
 	var/amount = 10
-	var/replacetiles = 0
-	var/eattiles = 0
-	var/maketiles = 0
-	var/fixfloors = 0
-	var/autotile = 0
-	var/nag_on_empty = 1
-	var/nagged = 0 //Prevents the Floorbot nagging more than once per refill.
+	/// Add tiles to existing floor
+	var/replacetiles = FALSE
+	/// Add floor tiles to inventory
+	var/eattiles = FALSE
+	/// Convert metal into floor tiles (drops on floor)
+	var/maketiles = FALSE
+	var/fixfloors = FALSE
+	/// Fix the floor and include a tile.
+	var/autotile = FALSE
```

Comment:
While we are here fixing stuff in this file, these variables should probably be converted to snake_case, to properly fit our style requirements.

Reply:
to be completly honest I have gotten very used to bots having their functions written like this, however if it is requested I will go ahead and change em, including other bots so they all look similar

Reply:
New commits, just need a double check to see they are fine.